app/views/users/passwords/new.html.erb

```diff
@@ -1,18 +1,19 @@
 <!--Descriptive page name, messages and instructions-->
 <h3 class="heading"><%= ts("Forgotten your password?") %></h3>
 <p>
-  <%= ts("If you've forgotten your password, we can send instructions that will allow you to reset it.") %>
-  <%= ts("Please tell us the user name or email address you used when you signed up for your Archive account.") %>
+  <%= ts("If you've forgotten your password, we can send you a temporary one. Please tell us the user name or email address you used when you signed up for your Archive account.") %>
+</p>
+<p>
+  <%= ts("If the temporary password doesn't work, please visit %{problems_when_logging_in} for help.", problems_when_logging_in: link_to(ts("Problems When Logging In"), admin_post_path("1277"))).html_safe %>
```

Comment:
I'm not sure the "Problems When Logging In" link should be there anymore, since the reset password process no longer involves logging in.

Maybe a link to Support, but I think that would belong more naturally on the `users/passwords/edit` page, since that's where the user will be when they run into errors trying to reset their password. (Unless they never get the email in the first place.)

Regardless, I think the issue on JIRA should be either closed or rewritten and repurposed.